cmd/main.go

```diff
@@ -368,12 +369,15 @@ func main() {
 
 	failOnError(autoMigrationWithLeader(autoMigrationLeader, db, log), "Failed auto migration process")
 
+	Options.EventsUploaderConfig.AssistedServiceVersion = versions.GetRevision()
+	uploadHandler := controllers.NewEventsUploaderWrapper(&Options.EventsUploaderConfig, db, log, ocpClient)
```

Comment:
I don't think that we always need it no?

Reply:
That's true. It's not needed for SaaS. I'll modify to create a dummy as you mentioned in this comment https://github.com/openshift/assisted-service/pull/4880#discussion_r1067999044